java/driver/jdbc-validation-derby/src/test/java/org/apache/arrow/adbc/driver/jdbc/derby/DerbyConnectionTest.java

```diff
@@ -28,4 +39,31 @@ public class DerbyConnectionTest extends AbstractConnectionTest {
   static void beforeAll() {
     quirks = new DerbyQuirks(tempDir);
   }
+
+  @Test
+  void newUrlOption() throws Exception {
+    try (final BufferAllocator allocator = new RootAllocator()) {
+      AdbcDriver driver = new JdbcDriver(allocator);
+      Map<String, Object> parameters = new HashMap<>();
+      parameters.put("url", "jdbc:derby:memory:newUrlOption;create=true");
```

Comment:
should we standardize on `"uri"` or make this a constant?

Reply:
Gah, I meant to have "uri" here yes. Thanks for catching me.

Reply:
Fixed - now uses uri